configure.php

```diff
@@ -1089,6 +1041,69 @@ function xinclude_residual( DOMDocument $dom )
     }
     if ( $see )
         echo "  See: https://github.com/php/doc-base/blob/master/docs/structure.md#xmlid-structure\n";
+
+    global $ac;
+    $fatal = $ac['LANG'] == 'en';
```

Comment:
```suggestion
    $fatal = $GLOBALS['ac']['LANG'] == 'en';
```
Please do all you can to avoid using globals, but if it's too much work to change it in this PR, please at least use `$GLOBALS` as this avoids nasty side effects and improves readability.